pkg/skaffold/tags/paths.go

```diff
@@ -58,22 +58,16 @@ func makeFilePathsAbsolute(config interface{}, base string) []error {
 				switch v.Interface().(type) {
 				case string:
 					path := v.String()
-					if path == "" {
-						return errs
-					}
-					if filepath.IsAbs(path) {
-						return errs
+					if path == "" || filepath.IsAbs(path) {
+						continue
```

Comment:
was erroneously returning here. That meant if two successive fields in a struct  have the `filepath` tag (like [HelmRelease](https://github.com/GoogleContainerTools/skaffold/blob/cb00826fd787ae1244c268e97a00a08a7bd82482/pkg/skaffold/schema/latest/config.go#L679-L682)) and the first is set to an absolute value already, then the second field did not get processed.